larray/core/session.py

```diff
@@ -455,31 +469,48 @@ def save(self, fname, names=None, engine='auto', overwrite=True, display=False,
 
         Examples
         --------
+        >>> # scalars
+        >>> i, s = 5, 'string'
         >>> # axes
-        >>> a, b = Axis("a=a0..a2"), Axis("b=b0..b2")    # doctest: +SKIP
+        >>> a, b = Axis("a=a0..a2"), Axis("b=b0..b2")
```

Comment:
there is a reason all these doctests were skipped! This is because they depend on pytables which is an optional dependency!

Reply:
I removed (temporarily) the `# doctest: +SKIP` just for testing the outputs of the examples. 
I simply forgot to put them back before to commit. Sorry.

Reply:
no problem then

Reply:
I didn't forget again about this. I will put back the `# doctest: +SKIP` just before to merge the PR.